### ISSUE TITLE
Show Link Previews

### ISSUE THREAD

**Issue by vachan-maker** [Nheko-Reborn/nheko#561]
Some people don't like this feature but I think this should be added to nheko

There are users who use GIFs from Tenor, Giphy, gyfcat, etc. They share the gifs or images as links. So when there are a lot of links being shared by the members in a room - its kind of tedious to open every link in the browser.

![2021-04-27 13_48_35-nheko](https://user-images.githubusercontent.com/65799568/116209225-5386b180-a75f-11eb-81d6-7f5a39819b44.png)

**Comment by LorenDB** [comment]
I agree, but I definitely think that this should be optional (which shouldn't be *too* hard...).

**Comment by McSinyx** [comment]
When this is done, if previews for arbitrary pages are also supported, please fall back to the standard `title` and `meta[name='description']` instead of solely relying on Open Graph or Dublin Core.

**Comment by deepbluev7** [comment]
If we implemented it, we would be using the Matrix server APIs for link previews, so we don't get a choice in that :)

**Comment by AppTester7** [comment]
I wish to extend the Previews option to include e2ee Rooms/DMs as well
